src/src.pro

```diff
@@ -9,7 +9,7 @@ unix:!macx: include(../unixconf.pri)
 
 QT += network xml
 
-macx|*-clang*: QMAKE_CXXFLAGS += -Wno-range-loop-analysis
+macx|*-clang*: QMAKE_CXXFLAGS_WARN_ON += -Wno-range-loop-analysis
```

Comment:
This is kinda abusing the QMAKE system but it works. @Chocobo1 you can make the change directly yourself without squashing my commit into yours. I don't mind.

Reply:
Nice, it works!
I've squashed your commit into mine.